pytest-embedded-idf/pytest_embedded_idf/dut.py

```diff
@@ -579,3 +579,27 @@ def flash_via_jtag(self):
 
         if self._meta:
             self._meta.set_port_app_cache(self.serial.port, self.app)
+
+
+class LinuxDut(Dut):
+    """
+    Dut class for Linux targets
+
+    Attributes:
+        serial (LinuxSerial): `LinuxSerial` instance
+    """
+
+    def __init__(
+        self,
+        serial,
+        **kwargs,
+    ) -> None:
+        self.serial = serial
+        super().__init__(**kwargs)
+
+    def write(self, data: t.AnyStr) -> None:
+        self.serial.write(to_bytes(data, '\n'))
+
+    def close(self):
```

Comment:
seems like we don't need the `close` here.

the destructure sequence is determined by the `yield` and the `fixture` sequence. i.e. the `serial` `close`/`terminate` would be called before `dut` `close` method automatically.